base/backend.Service.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
       the same node if possible.
   labels:
     deploy: sourcegraph
+    rbac-admin: non-escalated
```

Comment:
Should we use a different label like `sourcegraph-rbac-admin` as @daxmc99 noted would've helped a customer understand it more easily to be a label of ours and not one that e.g. Kubernetes interprets?

Reply:
i don't mind doing it. the problem is the label already exists

Reply:
By problem, do you mean we would have to rename it in files not modified here currently and you'd prefer doing it in a separate PR -- or that it introduces a problem because someone may be relying on it currently or something else?

Reply:
the latter. it has been there for a while, it's mentioned in our docs and readme. not sure if somebody relies on it

Reply:
Doesn't seem that important to me. We just need to mention it in the migrate.md docs that it has been renamed, so if you use it (most do not) you'll need to use the new name.

Reply:
ok, will do

Reply:
the deploy label stays the same, i assume ?

Reply:
Yeah, I would think so